crates/bindings-csharp/BSATN.Codegen/Utils.cs

```diff
@@ -131,8 +131,10 @@ static string GetTypeInfoForNamedType(INamedTypeSymbol type)
             var result = type.OriginalDefinition.ToString() switch
             {
                 // (U)Int128 are not treated by C# as regular primitives, so we need to match them by type name.
-                "System.Int128" => "SpacetimeDB.BSATN.I128",
-                "System.UInt128" => "SpacetimeDB.BSATN.U128",
+                "SpacetimeDB.Int128" => "SpacetimeDB.BSATN.I128",
+                "SpacetimeDB.UInt128" => "SpacetimeDB.BSATN.U128",
```

Comment:
Both need to be checked here, or it will break existing `System.Int128` support in C# modules.
```suggestion
                "System.Int128" or "SpacetimeDB.Int128" => "SpacetimeDB.BSATN.I128",
                "System.UInt128" or "SpacetimeDB.UInt128" => "SpacetimeDB.BSATN.U128",
```